products/sentinel_one.py

```diff
@@ -461,17 +471,28 @@ def _process_queries(self):
 
             for tag, queries in self._queries.items():
                 for query in queries:
-                    if query.operator in ('contains', 'containscis', 'contains anycase'):
+                    if query.operator in ('contains', 'containscis', 'contains anycase') and not re.findall(r'(?:\" AND)', query.search_value):
                         key = (query.operator, query.parameter)
                         if key not in combined_queries:
                             combined_queries[key] = list()
 
                         combined_queries[key].append((tag, query.search_value))
                     elif query.full_query is not None:
                         query_text.append((tag, query.full_query))
-                    else:
-                        full_query = query.parameter + ' ' + query.operator + ' ' + query.search_value
+                    elif query.operator == 'raw' and query.parameter == 'query':
+                        temp_search_value = query.search_value.lstrip('\"')
+                        if temp_search_value.endswith(')"'):
+                            temp_search_value = temp_search_value.rstrip('\"')
+                        full_query = f'({temp_search_value})'
                         query_text.append((tag, full_query))
+                    else:
+                        if re.findall(r'(?:\" AND)', query.search_value):
+                            for value in query.search_value.split(','):
+                                full_query = f'({query.parameter} {query.operator} {value})'
+                                query_text.append((tag, full_query))
+                        else:
+                            full_query = query.parameter + ' ' + query.operator + ' ' + query.search_value
+                            query_text.append((tag, full_query))
```

Comment:
No need to have this section since we already cover query fields above starting around line 482

```suggestion
```

Reply:
the second else statement is needed in case nothing matches the above checks. I have reformatted this to remove the regex check.